enslig-forsorger/src/main/kotlin/no/nav/familie/kontrakter/ef/søknad/Aksjeselskap.kt

```diff
@@ -0,0 +1,4 @@
+package no.nav.familie.kontrakter.ef.søknad
+
+class Aksjeselskap(navn: Søknadsfelt<String>,
+                   arbeidsmengde: Søknadsfelt<Int>)
```

Comment:
orgnummer?

Reply:
Ser att det ikke er med i skissene, men er det ikke bedre vi innhenter orgnummer i stedet for navn?

Reply:
Sjekker med Tuva :)